client/app/host/page.tsx

```diff
@@ -1,3 +1,476 @@
+                <span className="flex w-full justify-start pb-[10px]">
+                    <Dialog>
+                        <DialogTrigger className="flex items-center" asChild>
+                            <Button variant={'outline'} className="mr-[5px]">
+                                <MessageCirclePlus className="pr-[2px]" />
+                                Add Question
+                            </Button>
+                        </DialogTrigger>
+                        <DialogContent>
+                            <DialogHeader>
+                                <DialogTitle>New Question</DialogTitle>
+                                <DialogDescription>
+                                    Please enter all the required information for your question
+                                    below.
+                                </DialogDescription>
+                                {errorMessage && <p className="text-red-500">{errorMessage}</p>}
+                            </DialogHeader>
+                            <div>
+                                <Label htmlFor="question">Question</Label>
+                                <Input
+                                    id="question"
+                                    placeholder="Enter your question"
+                                    value={newQuestionText}
+                                    onChange={(e) => setNewQuestionText(e.target.value)}
+                                />
+                                <Label htmlFor="points">Points</Label>
+                                <Input
+                                    id="points"
+                                    placeholder="Enter points"
+                                    value={newQuestionPoints}
+                                    onChange={(e) => setNewQuestionPoints(e.target.value)}
+                                />
+                                <Select
+                                    value={selectedLanguage}
+                                    onValueChange={(value) => setSelectedLanguage(value)}
+                                >
+                                    <SelectTrigger className="mr-[5px] w-[180px]">
+                                        <SelectValue placeholder="Select a langauge ..." />
+                                        <SelectContent>
+                                            <SelectGroup>
+                                                <SelectItem value="rs">Rust</SelectItem>
+                                                <SelectItem value="py">Python</SelectItem>
+                                                <SelectItem value="java">Java</SelectItem>
+                                                <SelectItem value="js">JavaScript</SelectItem>
+                                            </SelectGroup>
+                                        </SelectContent>
+                                    </SelectTrigger>
+                                </Select>
```

Comment:
Need space between this and above element:
![image](https://github.com/user-attachments/assets/87e94276-73cf-4eec-bf0e-7f83f3cc5d4b)